transmart-copy/src/main/groovy/org/transmartproject/copy/table/Observations.groovy

```diff
@@ -36,6 +35,8 @@ class Observations {
     final Concepts concepts
     final Patients patients
 
+    private final Map<Long, Table> partitionToTable = [:]
```

Comment:
Why Long, not Integer?

Reply:
it's Long story ))

Reply:
The map contains DB identifiers as keys.